### DISCUSSION TITLE
About Note Templates

### DISCUSSION THREAD

**Discussion by windingwind** [windingwind/zotero-better-notes#219] [Note Templates]
This category lists note templates from the Zotero Better Notes community. Search the keyword of your need, like `tag`, `annotation`, or `table` to find the corresponding templates.
本文档罗列了由社区贡献的插件笔记模板。搜索关键词，例如文献条目（"Item"），笔记（"note"），标签（"tag"），表格（"table"）等。请善用浏览器的翻译与搜索功能。

[Documentation](https://github.com/windingwind/zotero-better-notes/blob/master/docs/about-note-template.md)

## Looking For Help?
Start a new discussion and describe what you need. It is recommended to provide a base template to modify with.

## How to Share Your Template
Please start a new discussion following the format:
Title: Your template name. Describe what this template does.

Body:
Copy the following markdown:
````md
Type: Item/Text/...

Description: This template is for...

Screenshots: ![img](upload images to show what this template looks like)

To import this template:
1. Copy the template below.
<details style="text-indent: 2em">
<summary>Click to show the example template</summary>

```yaml
name: "[TYPE] TEMPLATE NAME"
content: |-
  // @author YOUR NAME
  // @link PUBLISH PAGE URL
  TEMPLATE CONTENT HERE
```

</details>

2. Goto Zotero menubar, click `Edit`->`New Template from Clipboard`.
3. Click OK.
````

**Comment by dosomethingbyme** [comment]
# This template is specifically for importing/sharing, using better 
# notes 'import from clipboard': copy the content and
# goto Zotero menu bar, click Edit->New Template from Clipboard.  
# Do not copy-paste this to better notes template editor directly.
name: "[Item]论文笔记"
content: |-
  
  <h1 style="color:#193c47; background-color:#eef9fd; padding:8px;">
    ${(() => {
      const titleTranslation = topItem.getField("titleTranslation");
      return `(${topItem.getField("date")}) ${topItem.getField("title")}${titleTranslation ? ` (${titleTranslation})` : ''}`;
    })()}
  </h1>
  
  
  <table>
    
    <tr>
      <td style="color:#193c47; background-color:#dbeedd; padding:8px;">
        <b>作者:</b> ${topItem.getCreators().map((v) => `${v.firstName} ${v.lastName}`).join("; ")}
      </td>
    </tr>
  
    
    <tr>
      <td style="color:#193c47; background-color:#f3faf4; padding:8px;">
        <b>期刊:</b> <span style="color:#FF0000">${topItem.getField('publicationTitle')}</span> （发表日期: ${topItem.getField("date")}）
      </td>
    </tr>
  
    
    <tr>
      <td style="color:#193c47; background-color:#dbeedd; padding:8px;">
        <b>期刊分区:</b> ${(() => {
          const tags = Zotero.ZoteroStyle.api.renderCell(topItem, "publicationTags").childNodes;
          return Array.prototype.map.call(tags, e => e.outerHTML).join(' ');
        })()}
      </td>
    </tr>
  
    
    <tr>
      <td style="color:#193c47; background-color:#f3faf4; padding:8px;">
        ${(() => {
          const attachments = Zotero.Items.get(topItem.getAttachments());
          return attachments && attachments.length > 0
            ? `<b>本地链接:</b> <a href="zotero://open-pdf/0_${attachments[0].key}">${attachments[0].getFilename()}</a>`
            : `<b>本地链接:</b>`;
        })()}
      </td>
    </tr>
  
    
    <tr>
      <td style="color:#193c47; background-color:#dbeedd; padding:8px;">
        ${(() => {
          const doi = topItem.getField("DOI");
          return doi
            ? `<b>DOI:</b> <a href="https://doi.org/${doi}">${doi}</a>`
            : `<b>URL:</b> <a href="${topItem.getField('url')}">${topItem.getField('url')}</a>`;
        })()}
      </td>
    </tr>
    
    
    <tr>
      <td style="color:#193c47; background-color:#f3faf4; padding:8px;">
        <b>摘要:</b> <i>${topItem.getField('abstractTranslation') || topItem.getField('abstractNote')}</i>
      </td>
    </tr>
  
    
    <tr>
      <td style="color:#193c47; background-color:#dbeedd; padding:8px;">
        <b>笔记日期:</b> ${new Date().toLocaleString()}
      </td>
    </tr>
  
    
    <tr>
      <td style="color:#193c47; background-color:#f3faf4; padding:8px;">
        <b>上次编辑时间:</b> ${new Date().toLocaleString()}
      </td>
    </tr>
  </table>
  
  
  <h2 style="color:#66cdaa; background-color:#e0ffff;">📜 研究核心</h2>
  <blockquote style="color:gray; background-color:#f5f5f5; padding:8px;" contenteditable="false">提示: 研究内容，解决问题，创新点与不足。</blockquote>
  <h3>⚙️ 研究内容</h3>
  <blockquote>简述研究内容。</blockquote>
  <h3>📌 研究目标</h3>
  <blockquote>简述论文的研究目标。</blockquote>
  <h3>🔍 研究问题</h3>
  <blockquote>描述论文为解决什么问题而提出。</blockquote>
  <h3>✅ 研究成果</h3>
  <blockquote>说明论文解决了什么问题。</blockquote>
  
  <h2 style="color:#20b2aa; background-color:#afeeee;">🔁 研究过程</h2>
  <blockquote style="color:gray; background-color:#f5f5f5; padding:8px;" contenteditable="false">提示: 描述研究过程中的数据、方法、实验和结论。</blockquote>
  <h3>💧 数据</h3>
  <blockquote>描述使用的数据集。</blockquote>
  <h3>👩🏻‍💻 方法</h3>
  <blockquote>介绍研究的方法和技术。</blockquote>
  <h3>🔬 实验</h3>
  <blockquote>详细说明实验设计和结果。</blockquote>
  <h3>📜 结论</h3>
  <blockquote>总结研究的主要结论。</blockquote>
  
  <h2 style="color:#004d99; background-color:#87cefa;">🤔 个人总结</h2>
  <blockquote style="color:gray; background-color:#f5f5f5; padding:8px;" contenteditable="false">提示: 记录你对论文的总体评价、主要收获以及提出的疑问和改进建议。</blockquote>
  <h3>🌟 主要收获</h3>
  <blockquote>记录你从论文中获得的主要收获和重要见解。</blockquote>
  <h3>❓ 疑问与挑战</h3>
  <blockquote>列出你在阅读过程中遇到的疑问和认为存在的挑战。</blockquote>
  <h3>🛠 改进建议</h3>
  <blockquote>提出你认为可以改进的地方和方法。</blockquote>
  <h3>💭 思考与启发</h3>
  <blockquote>记录你从论文中得到的启发和对未来研究的思考。</blockquote>

**Reply by wenlyn-zyu** [reply]
sorry i don't understand. Zotero says this copy is invalid.
